12 - Key Sequence Detection/index-START.html

```diff
@@ -1,12 +1,29 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Key Detection</title>
-  <script type="text/javascript" src="https://www.cornify.com/js/cornify.js"></script>
-</head>
-<body>
-<script>
-</script>
-</body>
-</html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <title>Key Detection</title>
+      <script type="text/javascript" src="https://www.cornify.com/js/cornify.js"></script>
+    </head>
+    <body>
+      <script>
+        //// global variables
+        const secretSequence = 'pepperoni Pizza';
+        const currentSequence = [];
+        
+
+        //// callbacks
+        const keyDetector = (evt) => {
+          const badKeys = ['Shift', 'CapsLock', 'Dead', 'Control', 'Alt', 'Meta', 'Backspace'];
```

Comment:
interesting i also did not know about this text representation. interesting [read](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values)